<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>700</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tutorialButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>661</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>50</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #444;
border-radius: 8px;
padding: 4px;
color: white;
background-color: #222;</string>
   </property>
   <property name="text">
    <string>MTG Tutor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
